Package.swift

```diff
@@ -15,6 +15,7 @@ let package = Package(
       name: "TokamakDemo",
       targets: ["TokamakDemo"]
     ),
+    .executable(name: "RecRep", targets: ["RecRep"]),
```

Comment:
I'm not sure we should have this executable as part of the Tokamak package. It seems you have already adapted it to a test case, which should be enough.